Sources/ProjectDescription/RunAction.swift

```diff
@@ -34,6 +34,9 @@ public struct RunAction: Equatable, Codable {
     /// A target that will be used to expand the variables defined inside Environment Variables definition (e.g. $SOURCE_ROOT)
     public let expandVariableFromTarget: TargetReference?
 
+    /// The launch style of the action
+    public let launchStyle: LaunchStyle?
```

Comment:
And set the default in the init
```suggestion
    public let launchStyle: LaunchStyle
```